cg/src/domains/management/subjects/subjects.service.js

```diff
@@ -53,6 +53,39 @@ class SubjectsService {
 
     return decryptedSubjectsData;
   }
+
+  async listRectificationRequests(requestedPage) {
+    const page = requestedPage === undefined ? 1 : parseInt(requestedPage, 10);
+
+    const [{ total_items }] = await this.db('rectification_requests')
+      .where('status', 'PENDING')
+      .join('subject_keys', 'rectification_requests.subject_id', 'subject_keys.subject_id')
+      .select(this.db.raw('count(id) as total_items'))
+      .as('total_items');
+
+    const totalPages = Math.ceil(total_items / PAGE_SIZE || 1);
```

Comment:
for clarity the division could be enveloped in braces
```(total_items / PAGE_SIZE) || 1```

Reply:
prettier removes them

Reply:
Oh ok